### DISCUSSION TITLE
elevation/zIndex of views outside of a screen on Android

### DISCUSSION THREAD

**Discussion by benhowes** [software-mansion/react-native-screens#1083] [Q&A] [answered]
Hi,

I am not sure if I am seeing a bug, unexpected behaviour, or just simply not understanding what I am doing wrong! 

The situation:
- I have a `react-navigation` `bottom-tabs` navigator
- One of my navigation icons exceeds the bounds of the nav bar
- When `react-native-screens` is enabled, the overflowing button is hidden, but is fine without.

Expected (with `enableScreens(false);`):
![Screenshot_20210825-142149](https://user-images.githubusercontent.com/471170/130799868-4cd18683-c69d-4eb4-9f56-db154b5c2849.png)

Observed (with `enableScreens(true);`, `react-native-screens@3.4.0`):
![Screenshot_20210825-142226](https://user-images.githubusercontent.com/471170/130800026-22fe4d39-c057-46a4-9cd1-fd935e9d16bd.png)

I've done a lot of poking about in react-navigation and have extensively tested changing styles in that library without luck.

Is there a solution for allowing my menu button to overlap the `Screen`?

**Comment by tboba** [comment, marked as answer]
Hi @benhowes, sorry for really late response 😅 
Yeah, as what I can see this is more likely a bug. Do you know if that problem still occurs in the newest version of react-native-screens (which is currently `3.26.0`)? If so, please create an issue with steps to reproduce (including minimal repro - like the one you've attached above) 🙏 

Cheers!

**Reply by benhowes** [reply]
Thanks for the reply, but unfortunately I stopped working on this project nearly 2 years ago and cannot remember what I did with this in the end
